projects/plugins/jetpack/modules/likes/jetpack-likes-settings.php

```diff
@@ -653,13 +653,13 @@ public function is_comments_enabled() {
 	public function admin_settings_callback() {
```

Comment:
Should probably have something like this in here
```suggestion
	public function admin_settings_callback() {
		if ( ! isset( $_POST['_wpnonce'] ) || ! wp_verify_nonce( $_POST['_wpnonce'], 'sharing-options' ) ) {
			return;
		}

```
And then remove the nonce check down on line 691.